examples/tokenvm/tests/integration/integration_test.go

```diff
@@ -42,7 +43,6 @@ import (
 	"github.com/ava-labs/hypersdk/rpc"
 	"github.com/ava-labs/hypersdk/vm"
 
-	"github.com/ava-labs/hypersdk/auth"
```

Comment:
Not going to block on this but we need to fix this repo's imports. There doesn't seem to be any rationale for how the imports are ordered currently so I keep having to manually undo the diffs my ide auto-organizes.